### PR TITLE
Improvements to sentry-gateway

### PR DESCRIPTION
**Pull request by sagor999**
Hi @mikeroll 
here is PR with a list of my changes:
1. switch to go mods and add vendor folder
2. add support for passing any environment into sentry event
3. extract alert severity and pass it to sentry
4. add support for extracting sentry environment from alert label
5. update travis deployment

It is a lot of changes, but 90% of them are due to adding vendor folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
 
 alertmanager-sentry-gateway is a webhook gateway for [Alertmanager](https://github.com/prometheus/alertmanager). This gateway receives webhooks from Alertmanager and sends alert information as an event to [Sentry](https://sentry.io).
 
-## Install
+## Install 
 
 ### Just want the binary?
 
-Go to the [releases page](https://github.com/mikeroll/alertmanager-sentry-gateway/releases), find the version you want, and download the tarball file.
+Go to the [releases page](https://github.com/sagor999/alertmanager-sentry-gateway/releases), find the version you want, and download the tarball file.
```
Comment (mikeroll) [inline README.md:11]: If we're to merge this back, we'll need the original username in things like github urls and docker repos.
There's quite a number of such instances but I only mention this once here.

File: .travis.yml [hunk 2]
```diff
@@ -11,9 +13,10 @@ script:
   - task test
 before_deploy:
   - task dist
-deploy:
+deploy: 
   provider: releases
-  api-key: $GITHUB_TOKEN
+  api_key:
```
Comment (mikeroll) [inline .travis.yml:18]: Same as with usernames, no need to change this since we're merging back.

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 language: go
+dist: xenial
+os: linux
 go:
-  - "1.11.x"
+  - "1.13.x"
 before_install:
   - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```
Comment (mikeroll) [inline .travis.yml:7]: As I understand, `dep` is out of the picture now, so we better remove all mentions of it.

**Review by mikeroll — CHANGES_REQUESTED** [review]

**Comment by sagor999** [comment]
I am going to close this PR and re-open another one from a different branch. It will be easier for me to make any fixes that way.

**Pull request closed without merging**